lib/pool.js

```diff
@@ -249,6 +263,26 @@ class Pool extends EventEmitter {
   }
 }
 
+function addCachedSessionToTLSOptions (pool, options) {
+  let tls = options.tls
+
+  if (!tls) {
+    tls = {}
+  }
+
+  // User has chosen to opt out of TLS session reuse
+  if (tls.reuseSessions === false) {
```

Comment:
```suggestion
  if (!tls.reuseSessions) {
```

Reply:
Thanks for the suggestion @Ethan-Arrowood. As mentioned in the PR's description this strict equality check is on purpose.
Changing to your code will change the behavior of the pool to not reuse sessions by default.

Reply:
Oh I see now